crates/prune/src/parts/mod.rs

```diff
@@ -0,0 +1,123 @@
+mod receipts;
+pub(crate) use receipts::Receipts;
+
```

Comment:
```suggestion
```

Reply:
I added a whitespace so we don't mix up `pub`/`pub(crate)` exports with imports for the needs of `mod.rs`